app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
+    @reCheckIfInterventionNeeded()
```

Comment:
Here's how the API client creates an instance that's shared with any component that imports it, if that helps. https://github.com/zooniverse/panoptes-javascript-client/blob/master/lib/api-client.js#L4